plan_dinners.py

```diff
@@ -184,6 +184,17 @@ def shopping(self):
         )
         return self.shopping_df
 
+    def clear_sheet(self, range_name:str):
+        """Clear Shopping sheet before refilling with new data"""
+        try:
+            service = build("sheets", "v4", credentials=self.credentials)
```

Comment:
Use the same `result = (...)` and `return result` of the update_sheet function for consistency and the ability to use the result if needed in the future :)